languages/javascript/reference/info/immutability.md

```diff
@@ -0,0 +1,6 @@
+## Immutability
```

Comment:
Just a style question for us: Why is that a h2 (`##`) and not a h1 (`#`)?

I would think since that is the main concept in this file it should be a first degree headline.